docs/content/users/quickstart.md

```diff
@@ -149,6 +149,40 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
         ddev launch
         ```
 
+=== "ExpressionEngine"
+
+    ## ExpressionEngine
+
+    === "ExpressionEngine Zipfile Download"
+
+        Download the ExpressionEngine code from [ExpressionEngine.com](https://expressionengine.com/), then:
```

Comment:
Clarify we’re running things and link back to download+install instructions since they live in a different place for the Git install instructions:

```suggestion
        Download the ExpressionEngine code from [expressionengine.com](https://expressionengine.com/), then follow the [install instructions](https://docs.expressionengine.com/latest/installation/installation.html) by running the following:
```

Reply:
The odd thing here is that the DDEV instructions are far simpler and easier than the expressionengine.com instructions. You don't have to do most of the things when you use DDEV.